letta/client/client.py

```diff
@@ -68,6 +68,7 @@ def agent_exists(self, agent_id: Optional[str] = None, agent_name: Optional[str]
     def create_agent(
         self,
         name: Optional[str] = None,
+        agent_type: Optional[AgentType] = None,
```

Comment:
Could we have this explicitly default to the `AgentType.memgpt_agent`? Just to make it explicit what the defaults are

Reply:
Fixed